css/style.css

```diff
@@ -0,0 +1,51 @@
+.nav {
+  display: flex;
+  flex: 1 1 auto;
+  justify-content: space-between;
+  list-style-type: none;
+  text-align: center;
+  padding: 1em;
+}
```

Comment:
- [ ] I would like to advise that you stick to project requirements. One of the project requirements states that you follow the design template provided by Figma including the use of fonts, margins, and colors. Kindly include `font-family` of `Poppins` in this CSS declaration and use the `color` of `#6070ff`.

Reply:
I have changed the color to #6070ff, used the Poppin font family, and include the link in the HTML head section.